src/driver/Program.fs

```diff
@@ -208,6 +208,7 @@ module Fuzz =
             sprintf "--restler_grammar \"%s\"" parameters.grammarFilePath
             sprintf "--custom_mutations \"%s\"" parameters.mutationsFilePath
             sprintf "--set_version %s" CurrentVersion
+            sprintf "--client_certificate_path \"%s\"" parameters.certFilePath
 
             (match parameters.refreshableTokenOptions with
```

Comment:
>refreshableTokenOptions [](http://example.com/codeflow?start=30&length=23)

See comment in Types.fs - could you please make this parameter only be passed if the user sets the option?  

match parameters.certFilePath with
| None -> ""
| Some cfPath -> <your current sprintf>